README.md

```diff
@@ -1,8 +1,89 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Vishay Singh . Jarvis Consulting
 
+Hello there! My name is Vishay Singh and I graduated from the University of Toronto with an Honours 
+Bachelor's Degree of Science, majoring in Computer Science and French Studies. Since a young age, I've
+been fascinated by what technology can do to solve real world problems, which is exactly what I want 
+to do in the software industry. In fact, learning new things is a passion of mine, so software provides 
+the perfect avenue to continually challenge myself. In my spare time, I'm either building computers, 
+playing video games, or learning guitar.
```

Comment:
Lots of flavour, but need more technical details on your skills and experience. What projects have you worked on, what technologies have you used, what sorts of problems have you spent time on or like to solve, etc. This introduction should essentially include a highlights/summary of the rest of the profile.

Reply:
Take a look at these sample profiles for ideas (I do like the attitude and personal flair you express here though, try not to lose that)
https://github.com/jarviscanada/jarvis_data_eng_BrandonDicicco
https://github.com/jarviscanada/jarvis_data_eng_JohnCao
https://github.com/jarviscanada/jarvis_data_eng_TomasRotbauer
https://github.com/jarviscanada/jarvis_data_eng_TinaLee
https://github.com/jarviscanada/jarvis_data_eng_NarimanAlimuradov
https://github.com/jarviscanada/jarvis_data_eng_JasmeetBrar
https://github.com/jarviscanada/jarvis_data_eng_AlexKirilenko
https://github.com/jarviscanada/jarvis_data_eng_FatimaAnwer